WebNetSample/wwwroot/log.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Message": {
+    "MethodName": "GetProductDetailsAsync",
+    "LogParameters": [],
+    "FolderLocation": "C:\\Users\\Osman Ali\\Desktop\\WebNetSample\\WebNetSample"
```

Comment:
Hardcoding full file path is not a good idea since I will not be able to launch it and will see mistakes, let's make it relative

Reply:
I tried other ways but this time it doesnt work..